### PR TITLE
Update Changelog for 3.0.4

### PR DESCRIPTION
**Pull request by bricesanchez**
Please update and merge it after merging this PR : #3146

### CODE REVIEW

File: changelog.md [hunk 1]
```diff
@@ -3,12 +3,26 @@
 * [See full list](https://github.com/refinery/refinerycms/compare/3-0-stable...master)
 
 
+## 3.0.5 [unreleased]
+
+* [See full list](https://github.com/refinery/refinerycms/compare/3.0.4...master)
```
Comment (parndt) [inline changelog.md]: we should start comparing with `3-0-stable`

**Comment by bricesanchez** [comment]
Done. Could you write tests for https://github.com/refinery/refinerycms/pull/3146 :)

File: changelog.md [hunk 2]
```diff
@@ -1,14 +1,32 @@
-## 3.1.0 [unreleased]
+## 4.0.0 [unreleased]
+* Rails 5 doesn't allow Rubies before 2.2.2, let's not test them. [#3177](https://github.com/refinery/refinerycms/pull/3177). [Philip Arndt](https://github.com/parndt)
```
Comment (parndt) [inline changelog.md]: this is CI, not really worthy of a changelog

Comment (bricesanchez) [inline changelog.md]: fixed

**Comment by bricesanchez** [comment]
The branch is rebased and waiting for CI

**Pull request merged**